docs/spec/attributes.decoder.md

```diff
@@ -5,7 +5,7 @@
 
 ~~~~~
 DecodeAttributesDecoderData(buffer) {
-  num_attributes                                                        I32
+  num_attributes                                                        varUI32
```

Comment:
@ondys, I'm going to merge this. Could you open an issue to track this concern? Thanks.